src/components/ModelizerModelView.vue

```diff
@@ -141,14 +165,52 @@ async function updatePluginsAndTemplates() {
       templates.value = response;
     })
     .catch(() => {
-      Notify.create({
-        type: 'negative',
-        message: t('errors.templates.getData'),
-        html: true,
-      });
+      notifyError();
     });
 }
 
+/**
+ * Instantiate from a dragged component definition or template.
+ * @param {DragEvent} event - The drag event.
+ * @return {Promise<void>} - Implicitly declared Promise wrapping the function.
+ */
+async function dropHandler(event) {
+  const dropData = JSON.parse(event.dataTransfer.getData('text/plain'));
+  const modelFolder = process.env.MODELS_DEFAULT_FOLDER
+    ? `${process.env.MODELS_DEFAULT_FOLDER}/${route.query.path}`
+    : `${route.query.path}`;
+
+  let files;
+
+  if (dropData.isTemplate) {
+    const activeTemplate = templates.value.find(
+      (template) => template.plugin === dropData.pluginName
```

Comment:
Remove this line, because, we only have template for the active plugin related to the dragged component.
update line like this: 
```
({key}) => key === dropData.definitionType,
```

Reply:
done